leo/cli/commands/devnode/rest/mod.rs

```diff
@@ -0,0 +1,337 @@
+            // Get ../consensus_version
+            .route("/consensus_version", get(Self::get_consensus_version))
+
+            // GET ../block/..
+            .route("/block/height/latest", get(Self::get_block_height_latest))
+            .route("/block/hash/latest", get(Self::get_block_hash_latest))
+            .route("/block/latest", get(Self::get_block_latest))
+            .route("/block/{height_or_hash}", get(Self::get_block))
+            // The path param here is actually only the height, but the name must match the route
+            // above, otherwise there'll be a conflict at runtime.
+            .route("/block/{height_or_hash}/header", get(Self::get_block_header))
+            .route("/block/{height_or_hash}/transactions", get(Self::get_block_transactions))
+            .route("/block/create", post(Self::create_block))
+
+            // GET and POST ../transaction/..
+            .route("/transaction/broadcast", post(Self::transaction_broadcast))
+            .route("/transaction/confirmed/{id}", get(Self::get_confirmed_transaction))
+            .route("/transaction/{id}", get(Self::get_transaction))
+
+            // GET ../find/..
+            .route("/find/blockHash/{tx_id}", get(Self::find_block_hash))
+            .route("/find/blockHeight/{state_root}", get(Self::find_block_height_from_state_root))
+            .route("/find/transactionID/deployment/{program_id}", get(Self::find_latest_transaction_id_from_program_id))
+            .route("/find/transactionID/deployment/{program_id}/{edition}", get(Self::find_transaction_id_from_program_id_and_edition))
+            .route("/find/transactionID/{transition_id}", get(Self::find_transaction_id_from_transition_id))
+            .route("/find/transitionID/{input_or_output_id}", get(Self::find_transition_id))
+
+            // GET ../program/..
+            .route("/program/{id}", get(Self::get_program))
+            .route("/program/{id}/latest_edition", get(Self::get_latest_program_edition))
+            .route("/program/{id}/{edition}", get(Self::get_program_for_edition))
+            .route("/program/{id}/mappings", get(Self::get_mapping_names))
+            .route("/program/{id}/mapping/{name}/{key}", get(Self::get_mapping_value))
+            .route("/program/{id}/mapping/{name}", get(Self::get_mapping_values))
+
+            // GET misc endpoints.
+            // .route("/version", get(Self::get_version))
+            .route("/blocks", get(Self::get_blocks))
+            .route("/height/{hash}", get(Self::get_height))
+            .route("/statePath/{commitment}", get(Self::get_state_path_for_commitment))
+            .route("/statePaths", get(Self::get_state_paths_for_commitments))
+            .route("/stateRoot/latest", get(Self::get_state_root_latest))
+            .route("/stateRoot/{height}", get(Self::get_state_root));
+
+        // If the `history` feature is enabled, enable the additional endpoint.
+        #[cfg(feature = "history")]
```

Comment:
I think we can get rid of "history". It's used to dump out historical mapping values and can bloat the disk.